### ISSUE TITLE
Proposal

### ISSUE THREAD

**Issue by Acylation** [Acylation/DLSilicaSpheres#2]
4.14, 5 min PPT

- [x] 确认是否需要写开题文书？400~500 words

**Comment by Acylation** [comment]
## 课题背景

- http://www.dxhx.pku.edu.cn/CN/10.3866/PKU.DXHX202210039
- 其他项目资料，包括开题、中期、结题、竞赛材料，实验记录和数据

## 相关工作

- by256/bpartis
- matterport/Mask_RCNN
- BlendMask
- YOLACT
- SOLO
- FCIS
- deepimagej/deepimagej-plugin

## 技术选型

- 数据集标注
  - 格式：
  - 平台：CVAT/LabelMe
  - 模拟数据集的读取
- 前处理
  - ROI
  - 数据增强
  - 降噪
- 框架：Tensorflow
- 网络：（是否自行调整？）
  - Mask R-CNN
  - BlendMask
  - Bayesian
- Baseline
  - HoughCircle by opencv-cpp or opencv-python
  - ImageJ 的 Hough 模块
  - BPartis
  - HRNet/SegAnything
- Metrics
  - 识别率（按对象个数），尤其关注传统方法无法识别而DL可以识别的
  - 像素准确度 / 圆心、粒径拟合准确度
  -  sklearn库
  - 混淆矩阵
  - ROC曲线
  - 逐实例/逐像素？看看统计相关
  - ……

## 实施计划

- 数据集标注
- 计算资源（远程提交任务）
- 指标

## 阶段进展

## Review

将本文档中的内容迁移到各个具体内容中去
中期/结项报告可以复用

**Comment by Acylation** [comment]
Due 4/13 11:59pm

The project proposal should be one paragraph (200-400 words). Your project proposal should describe:

What is the problem that you will be investigating? Why is it interesting?

What reading will you examine to provide context and background?

What data will you use? If you are collecting new data, how will you do it?

What method or algorithm are you proposing? If there are existing implementations, will you use them and how? How do you plan to improve or modify such implementations? You don't have to have an exact answer at this point, but you should have a general sense of how you will approach the problem you are working on.

How will you evaluate your results? Qualitatively, what kind of results do you expect (e.g. plots or figures)? Quantitatively, what kind of analysis will you use to evaluate and/or compare your results (e.g. what performance metrics or statistical tests)?

Submission: Please submit your proposal as a PDF on 雨课堂 . Only one person on your team should submit. Please have this person add the rest of your team as collaborators as a "Group Submission".
